e2e/fixtures/cluster_config.go

```diff
@@ -135,6 +142,10 @@ func (config *ClusterConfig) SetDefaults(factory *Factory) {
 			log.Println("FoundationDBCluster", fdbCluster.Name(), "successfully created in", fdbCluster.Namespace())
 		}
 	}
+
+	if config.cloudProvider == "" {
```

Comment:
I need some additional logging here. Seems like the cloud provider his not picked up correctly.